crates/ruff/src/rules/flake8_todos/rules.rs

```diff
@@ -400,6 +389,7 @@ fn static_errors(
                 trimmed.text_len()
             }
         } else {
+            // TD-002
```

Comment:
Not directly related to this PR, but I didn't feel like it was worth raising a new one entirely.